src/Models/Constants/Types.ts

```diff
@@ -125,6 +125,11 @@ export type AdapterMethodParamsForSearchADTTwins = {
     continuationToken?: string;
 };
 
+export type AdapterMethodParamsForAdvancedSearchADTwins = {
+    query: string;
+    continuationToken?: string;
```

Comment:
I would rather the components not need to worry about this. How do you plan to handle triggering the next page?

Reply:
I thought we talked about including `top 1000` in the query. Then we could have the adapter simply keep resolving the pages until it hits the end.